src/com/google/enterprise/adaptor/documentum/DocumentumAdaptor.java

```diff
@@ -903,8 +913,20 @@ protected boolean fillCollection(IDfSession dmSession,
     @Override
     protected Checkpoint pushCollection(DocIdPusher pusher)
         throws InterruptedException {
-      pusher.pushGroupDefinitions(groups.build(),
-          caseSensitivityType == CaseSensitivityType.EVERYTHING_CASE_SENSITIVE);
+      FeedType feedType;
+      if ((queryBatchSize > 0) || caughtException || prevError) {
```

Comment:
Why || prevError?

Reply:
When there was an exception previously and no exception second time, the caughtException becomes false and feedType becomes this.feedType. This can be FeedType.FULL.

Reply:
What's wrong with that? We aren't using batching, and there was no error, so we have all the groups and a FULL feed. Why switch back to incremental again?